src/async.rs

```diff
@@ -251,6 +251,11 @@ impl<'a, T> FusedFuture for SendFut<'a, T> {
 pub struct SendSink<'a, T>(SendFut<'a, T>);
 
 impl<'a, T> SendSink<'a, T> {
+    /// Returns a clone of a sending half of the channel of this sink.
+    pub fn sender(&self) -> Sender<T> {
+        (*self.0.sender).clone()
+    }
```

Comment:
Why not `&self -> &Sender<T>`?

Reply:
That makes sense, I'll change it to that